packages/bits/src/lib/repeat/repeat-item/repeat-item.component.ts

```diff
@@ -21,4 +22,8 @@ export class RepeatItemComponent {
     @Input() public nowrap = false;
 
     @Input() public selected = false;
+
+    @Input() public draggable = false;
+
+    get role(): string { return this.clickable || this.draggable ? "option" : "listitem"; }
```

Comment:
I was wrong here, because of using "option" for draggable elements. Will fixed.

But we have enough information about the clickable and selectable items. 
![image](https://user-images.githubusercontent.com/75837119/113850609-536c4500-97a3-11eb-9f93-68cfa7fac3af.png)
 It seems like one condition isItClickable will be enought for us.